src/main/java/com/users/demo_users/DemoUsersApplication.java

```diff
@@ -0,0 +1,14 @@
+package com.users.demo_users;
```

Comment:
general convention is to package application based on stakeholder.

org/user
org.springframework.* or com.google.*

in case of single developers: {countryCode}.{yourNameNickOrAnythingLinkingToYou}.{basePackageForYourApp}
in my case it would be pl.konradboniecki.demousers.*